user_test.go

```diff
@@ -25,101 +25,39 @@ func TestUserClaims(t *testing.T) {
 	t.Logf("Token: %s", token)
 }
 
-func TestAppMetadataAuthorizationRoles(t *testing.T) {
```

Comment:
@njones , please sync with the latest commit. This one is my "app_metadata" change.